tests/expected/uninit/fixme_unions.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright Kani Contributors
```

Comment:
Is this test intentionally missing an expected file? If so, please add a TODO comment to add an expected file and rename the test once the necessary support is added.